index.js

```diff
@@ -10,15 +10,24 @@ var requiredOptions = [
     'handleMessage'
   ];
 
+/**
+ * Construct a new SQSError
+ */
+function SQSError(message) {
+  this.name = 'SQSError';
+  this.message = (message || '');
+}
+SQSError.prototype = Error.prototype;
+
 function validate(options) {
   requiredOptions.forEach(function (option) {
     if (!options[option]) {
-      throw new Error('Missing SQS consumer option [' + option + '].');
+      throw new SQSError('Missing SQS consumer option [' + option + '].');
```

Comment:
I think these can remain as just plain `Error`'s as they're thrown and not handled by any events.